D2IQ-81520: KM2.1 migration cleanup missing a command (#3963)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/cleanup/index.md

<!-- markdownlint-disable MD0013 MD030 -->

> After upgrade is complete, some Kubernetes objects that belonged to Konvoy 1.8 addons, but are not used by Kommander 2.1 Platform Applications may remain on the cluster. The existence of these objects has no impact on the function of Kommander 2.1. However, modifying or deleting them can corrupt the Kommander 2.1 installation produced by the upgrade. As a result, you should remove these objects using the following steps.

Reviewer: ```suggestion
After you complete the upgrade, some Kubernetes objects that belonged to Konvoy 1.8 addons, but are not used by Kommander 2.1 Platform Applications, could remain on the cluster. The existence of these objects has no impact on the function of Kommander 2.1. However, modifying or deleting these objects might corrupt the Kommander 2.1 installation produced by the upgrade. As a result, you should remove these objects using the following procedure:
```

## Components of the 1.8 logging stack

> After a successful upgrade, the 1.8 addons no longer exist. Helm releases of Kibana, Elasticsearch, and Elasticsearch-Curator remain and are not managed by the addons, if they were installed.

Reviewer: ```suggestion
After a successful upgrade, the 1.8 addons no longer exist. Helm releases of Kibana, Elasticsearch, and Elasticsearch-Curator remain but are not managed by the addons, if they were installed.
```

> Uninstall these Helm releases from the command line, using this Helm command:

Reviewer: ```suggestion
Uninstall these Helm releases from the command line, using the command:
```

> 

Reviewer 2: I'm not sure what happened here, but the output on line 28 (i.e. just before this line) is missing a line.

 We are directing the customer to uninstall 4 things:  kibana-kubeaddons, elasticsearch-curator-kubeaddons, elasticsearch-kubeaddons, elasticsearchexporter-kubeaddons

But the output section only lists three.   It's missing a line for 'kibana-kubeaddons'.   So it should look like this:

release "kibana-kubeaddons" uninstalled
release "elasticsearchexporter-kubeaddons" uninstalled
release "elasticsearch-curator-kubeaddons" uninstalled
release "elasticsearch-kubeaddons" uninstalled

## Istio

> If Istio is installed on the cluster, the upgrade leaves the Helm Secret that belonged to the 1.8 Istio addon in the `istio-system` namespace.

Reviewer: ```suggestion
If Istio is installed on the cluster, the upgrade retains the Helm Secret that belonged to the 1.8 Istio addon in the `istio-system` namespace.
```

> <p class="message--note"><strong>WARNING: </strong>As long as this secret exists, <code>helm list -n istio-system</code> will incorrectly report existence of an "istio-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Istio Platform Application in Kommander, which can disrupt operation of Istio workloads.</p>

Reviewer: ```suggestion
<p class="message--note"><strong>WARNING: </strong>As long as this secret exists, <code>helm list -n istio-system</code> incorrectly reports the existence of an "istio-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Istio Platform Application in Kommander, which can disrupt operation of Istio workloads.</p>
```

Delete this secret using the `kubectl` command:

```sh
kubectl delete -n istio-system secret sh.helm.release.v1.istio-kubeaddons.v1
```

## Gatekeeper

If Gatekeeper is installed on the cluster, the upgrade leaves a Helm Secret, Deployments, and a Service that belonged to the 1.8 Gatekeeper addon in the `kubeaddons` namespace.

> <p class="message--note"><strong>WARNING: </strong>As long as the secret exists, `helm list -n kubeaddons` will incorrectly report the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>

Reviewer: ```suggestion
<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, `helm list -n kubeaddons` incorrectly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```

```

<p class="message--note"><strong>WARNING: </strong>As long as the secret exists, <code>helm list -n kubeaddons</code>

> will wrongly report existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it may corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>

Reviewer: ```suggestion
wrongly reports the existence of a "gatekeeper-kubeaddons" Helm release. If you manipulate this "Helm release" using Helm, it might corrupt the Gatekeeper Platform Application in Kommander, potentially making Kubernetes on the cluster inoperable.</p>
```
